tests/src/test_helpers/ssl_helpers.c

```diff
@@ -2543,6 +2543,9 @@ int mbedtls_test_get_tls13_ticket(
                                          server_options, NULL, NULL, NULL);
     TEST_EQUAL(ret, 0);
 
+    mbedtls_ssl_conf_enable_new_session_tickets(
+        &client_ep.conf, MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_ENABLED);
```

Comment:
Test coverage: as of 0e5d4fdfc56540ecc23809f006e83f5cea3c26a5:

* Unit tests have the NewSessionTickets error enabled.
* All the ssl-opt test cases have the NewSessionTickets error enabled.
* `ssl_client1.c` has the NewSessionTickets error disabled, but does not run on the CI.

I'm ok with this as an emergency fix for 3.6.1. For 3.6.2, we must have proper test coverage. At the very least, we must merge https://github.com/Mbed-TLS/mbedtls/pull/9505, which will give us coverage of NewSessionTickets error disabled on the CI. Preferably we should also have unit tests for this setting. It's a setting added to an LTS in a hurry, so good test coverage is especially important to make sure we don't accidentally screw it up more.

Reply:
I've added two ssl-opt.sh tests with NewSessionTicket handling disabled at runtime similar to the two ones we have for tickets being disabled at build time.